scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -28,25 +28,28 @@ import scala.concurrent.Future
 import scala.reflect.ClassTag
 
 /**
- * Main package for JDBC APIs. Import all.
- *
- * {{{
- * import com.spotify.scio.jdbc._
- * }}}
- */
+  * Main package for JDBC APIs. Import all.
+  *
+  * {{{
+  * import com.spotify.scio.jdbc._
+  * }}}
+  */
 package object jdbc {
 
   type JdbcIO[T] = jdbc.nio.JdbcIO[T]
   val JdbcIO = jdbc.nio.JdbcIO
 
+  private[jdbc] val USE_BEAM_DEFAULT_BATCH_SIZE = -1L
```

Comment:
I wouldn't go crazy as de-magic-fy these since `-1` is a pretty common pattern but that's just me ;)

Reply:
thought it is more informative what actually -1 means here.